src/lib/serviceRoutinesV2/logLevelTreat.cpp

```diff
@@ -55,11 +55,44 @@ std::string changeLogLevel
   std::string  level   = ciP->uriParam[URI_PARAM_LEVEL];
   const char*  levelP  = level.c_str();
 
-  if (level.empty())
+  std::string logSize     = ciP->uriParam[URI_PARAM_LOG_SIZE];
+  int logSizeValue        = atoi(logSize.c_str());
+  std::string payloadSize = ciP->uriParam[URI_PARAM_PAYLOAD_SIZE];
+  int payloadSizeValue    = atoi(payloadSize.c_str());
+
+  if (level.empty() && logSize.empty() && payloadSize.empty())
   {
     ciP->httpStatusCode = SccBadRequest;
-    alarmMgr.badInput(clientIp, "no log level in URI param");
-    return "{\"error\":\"log level missing\"}";
+    alarmMgr.badInput(clientIp, "no log configs in URI param");
+    return "{\"error\":\"log config missing\"}";
+  }
+
+  if (!logSize.empty())
+  {
+    if (logSizeValue > 0)
+    {
+      logLineMaxSize = logSizeValue;
+    }
+    else
+    {
+      ciP->httpStatusCode = SccBadRequest;
+      alarmMgr.badInput(clientIp, "invalid logLineMaxSize in URI param", logSize);
+      return "{\"error\":\"invalid logLineMaxSize, logLine size should be > 0\"}";
+    }
+  }
+
+  if (!payloadSize.empty())
+  {
+    if (payloadSizeValue > 0)
+    {
+      logInfoPayloadMaxSize = payloadSizeValue;
+    }
+    else if (payloadSizeValue < 0)
+    {
+      ciP->httpStatusCode = SccBadRequest;
+      alarmMgr.badInput(clientIp, "invalid logInfoPayloadMaxSize in URI param", payloadSize);
+      return "{\"error\":\"invalid logInfoPayloadMaxSize,logPayload size should be > 0\"}";
```

Comment:
```suggestion
      return "{\"error\":\"invalid logInfoPayloadMaxSize, logPayload size should be > 0\"}";
```

(Changes in .test would be also required)

Reply:
Fixed in [46582d5](https://github.com/telefonicaid/fiware-orion/pull/4289/commits/46582d5125caad31b3df9116faa6797957652b95)